Character.cpp

```diff
@@ -98,20 +106,44 @@ void Character::attackInternal(Character& other)
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
       */
-        assert(false);
-        std::cout << getName() << " defeated " << other.getName() << " and leveled up!" << std::endl;        
+        initialStats(hitPoints, *initialHitPoints);
+        initialStats(armor, *initialArmorLevel);
+        initialStats(attackDamage, *initialAttackDamage);
```

Comment:
Name things what they are, name functions what they do.
`initialStats` is a thing.  it doesn't say what it does.
- [ ] revise